docs/bok/Activities/Level-3/Public-Development.md

```diff
@@ -8,32 +8,59 @@ sidebar_position: 1
 sidebar_label: Public Development
 ---
 
-**THIS IS A PLACEHOLDER**
+## Public Development
 
+### Working with others
 
-something about using GitHub or GitLab.
+The biggest thing to remember about open source development is that it happens in the public sphere, in written text, with other humans. Text can be a challenging medium in which to interact with others. Tone and intent can be hard to communicate - emoji help. Language barriers often exist. For this reason it is important open source developers strive for clarity, compassion, and empathy in their communication.
 
-Risk of DLP. (we have articles already on this, we need to reference)
+Many folks have written in depth on this topic. The following articles dig a bit deeper into the human side of open, collaborative development:
 
-small PRs.
+* [Why the success of open source depends on empathy](https://opensource.com/article/21/2/open-source-empathy), by Bronagh Sorota
 
```

Comment:
Relevant Linux Foundation training regarding OSS:

- [Inclusive Open Source Community Orientation (LFC102)](https://trainingportal.linuxfoundation.org/courses/inclusive-open-source-community-orientation-lfc102)
- [Inclusive Strategies for Open Source (LFC103)](https://trainingportal.linuxfoundation.org/courses/inclusive-strategies-for-open-source-lfc103)